src/components/HacksPage.js

```diff
@@ -22,18 +24,19 @@ const useStyles = makeStyles(theme => ({
   }
 }));
 
-const Hacks = () => {
+const Hacks = ({ dispatch }) => {
   const classes = useStyles();
   const [data, setData] = useState([]);
 
   useEffect(() => {
     const fetchData = async () => {
-      let result = await axios.get(process.env.REACT_APP_API_URL+"hacklist");
+      let result = await axios.get(process.env.REACT_APP_API_URL + "hacklist");
       //console.log(result.data);
       setData([...result.data]);
     };
     fetchData();
-  }, []);
+    dispatch(fetchUsers());
```

Comment:
Yeah the reason I used that is because the re render issue, I wasn't able to put the props.fetchUsers into the useEffect dependency array without causing infinite re renders, I used the temporary solution, another way was to disable eslint which I didn't think would be a good idea.

https://github.com/facebook/create-react-app/issues/6880#issuecomment-488158024

Reply:
That is very interesting, and not something I've come across before. I haven't using redux with hooks (I've used both redux and hooks separately).

It may also be worth noting the hooks from redux, `useDispatch` as perhaps a more preferred way to use redux and hooks together.

https://react-redux.js.org/api/hooks